ansible/roles/eos/templates/t1-64-lag-clet-tor.j2

```diff
@@ -1 +1 @@
-t1-64-lag-tor.j2
\ No newline at end of file
+t1-lag-spine.j2
```

Comment:
Why use role `spine` of t1-lag for role `tor` of t1-64-lag-clet?
Does `t1-64-lag-clet-tor.j2` need to be updated as well?

Reply:
Let's move away from clet configuration for now. This configuration is not 100% right.

Reply:
t1-64-lag-clet-tor.j2 -> t1-64-lag-tor.j2 the clet configurations are symbolic links, they are fine. But strange diff show up here.

Reply:
Why changing this symbolic link?

Reply:
This has now been corrected in the latest commit. No changes for the symbolic link of `t1-64-lag-clet-tor.j2`. The spine template for `t1-64-lag-clet-spine.j2` is changed however to point to `t1-lag-spine.j2` instead of `t1-64-lag-spine.j2` (as this is now a sym link itself). Thanks.